File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/com/github/asyncmc/mojang/sessions/scala/gatling/model/PlayerSkinMetadata.scala

package com.github.asyncmc.mojang.sessions.scala.gatling.model


case class PlayerSkinMetadata (
    /* The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent. */
    _model: Option[String]
)
object PlayerSkinMetadata {
    def toStringBody(var_model: Object) =
        s"""
        | {
        | "model":$var_model
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-sessions/src/main/kotlin/com/github/asyncmc/mojang/sessions/kotlin/spring/model/PlayerSkinMetadata.kt
package com.github.asyncmc.mojang.sessions.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonValue
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Contains details about the skin
 * @param model The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent.
 */
data class PlayerSkinMetadata (

        @JsonProperty("model") val model: PlayerSkinMetadata.Model? = null
) {

    /**
    * The player model, currently only \"slim\" (Alex) is valid, for Steve's model this property must be absent.
    * Values: SLIM
    */
    enum class Model(val value: String) {
    
        @JsonProperty("slim") SLIM("slim");
    
    }

}

